python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_multioutput_firesatellite.py

```diff
@@ -83,29 +83,16 @@
 
 # %%
 scaleOptimizationBounds = ot.Interval(
-    [1.0, 1.0, 0.1, 0.01, 0.1, 0.1, 0.01, 0.01, 0.001, 0.01, 0.01, 0.01],
-    [1.0e7, 2.0e3, 2.0e3, 1e2, 10.0, 10.0, 10.0, 10.0, 10.0, 1e8, 1e4, 1e3],
+    [1.0e6, 1.0e3, 1e3, 1., 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
+    [2.0e7, 2.0e3, 2.0e3, 1e2, 10.0, 10.0, 10.0, 10.0, 10.0, 10, 10, 10],
```

Comment:
```suggestion
    [1.0e6, 1.0e3, 1.0e3, 1.0, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
    [2.0e7, 2.0e3, 2.0e3, 1e2, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0],
```
For the sake of consistency